nomos-services/data-availability/network/src/backends/libp2p/common.rs

```diff
@@ -75,6 +78,13 @@ impl SamplingEvent {
     }
 }
 
+#[derive(Debug)]
+pub enum PeerRequest {
```

Comment:
Just idea, could this be `MonitorCommand` as it goes directly to the monitor behaviour?